Client/Frontend/Browser/BrowserViewController/BrowserViewController+ProductNotification.swift

```diff
@@ -10,6 +10,45 @@ import Shared
 // MARK: - ProductNotification
 
 extension BrowserViewController {
```

Comment:
I like that you moved this code out of BVC class to not bloat it further :)

Reply:
Right now this logic presides in a separate extension and file but If we want to do a further separation we should think about changes in design for BrowserViewController